WineCellar/WineCellar.GUI/RegisterWine.xaml

```diff
@@ -47,13 +53,13 @@
                 </StackPanel>
 
                 <StackPanel Orientation="Horizontal">
-                    <Label Content="Inkoopprijs"/>
+                    <Label Content="Inkoopprijs (€)&#xD;&#xA;"/>
```

Comment:
Heeft deze encoded tekst (dat wat in de property "Content" staat) een functie? Zo niet, graag weghalen om leesbaarheid te verbeteren.

Reply:
Fixed